### ISSUE TITLE
Why the output of the CTC decoder is not seperated by space?

### ISSUE THREAD

**Issue by dreamibor** [PaddlePaddle/DeepSpeech#244]
Hi,

I used logits from another model (Wav2Letter) to test the CTC decoder, however, I can't get correct results from the decoder, for example,
I got :"sometimemesttheeultbotthfehinatetwhorkhasreallygod" from the decoder while the greedy decoder gave me "sometimd es the ou but of fan at work is really god".
I am wondering why the CTC decoder will concatenate all the characters?

Thanks,

**Comment by dreamibor** [comment]
Haha, it turns out I have to put a softmax layer between the logits and the CTC decoder to normalise the input.
